libs/dkg/src/dkg_complaints_manager.cpp

```diff
@@ -18,17 +18,14 @@
 
 #include "dkg/dkg_complaints_manager.hpp"
 
-#include <cstdint>
-#include <mutex>
-
 namespace fetch {
 namespace dkg {
 
 constexpr char const *LOGGING_NAME = "DKGComplaints";
 
 void ComplaintsManager::ResetCabinet(uint32_t cabinet_size)
 {
-  FETCH_LOCK(mutex_);
+  std::lock_guard<std::mutex> lock{mutex_};
```

Comment:
This looks like a retrograde step?

Reply:
I don't think I have been changing these around - @jinmannwong ?

Reply:
Not me either. I was confused by this too.

Reply:
reverted to FETCH_LOCK